### PR TITLE
fix: audit 12.09

### PR DESCRIPTION
**Pull request by nksazonov**

### CODE REVIEW

File: src/interfaces/IVault.sol [hunk 1]
```diff
@@ -1,11 +1,16 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+pragma solidity 0.8.26;
```
Comment (mod) [inline src/interfaces/IVault.sol]: I would not lock interfaces as many components could include with different version of solidity

Comment (nksazonov) [reply src/interfaces/IVault.sol]: b4f68b66027810ab7280cf5bcb9d86532c9dd9e3

File: test/BrokerToken.t.sol [hunk 2]
```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.24;
```
Comment (maxpushka) [inline test/BrokerToken.t.sol]: Previously you mentioned that strict version is used for contract implementation and floating version for interfaces. What about tests?

File: test/TestERC20.sol [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity ^0.8.22;
```
Comment (maxpushka) [inline test/TestERC20.sol]: Previously you mentioned that strict version is used for contract implementation and floating version for interfaces. What about tests?

**Review by maxpushka — APPROVED** [review]

File: test/BrokerToken.t.sol (hunk 2 above)
Comment (nksazonov) [reply test/BrokerToken.t.sol]: 9114873cf19ef71dc69a31376e3fb093107b061a

File: test/TestERC20.sol (hunk 3 above)
Comment (nksazonov) [reply test/TestERC20.sol]: 9114873cf19ef71dc69a31376e3fb093107b061a

**Pull request merged**